.travis.yml

```diff
@@ -13,7 +13,7 @@ cache:
 # FIXME: replace with cmdstanr master eventually
 r_github_packages:
   - r-lib/covr
-  - stan-dev/cmdstanr@install_cmdstan_from_r
+  - stan-dev/cmdstanr
```

Comment:
What's this line?

Reply:
Travis was looking for the wrong branch. Switched to master. Unrelated to this PR but travis will fail without this fix.